src/Microsoft.VisualStudio.Editor.Razor/VisualStudioRazorParser.cs

```diff
@@ -11,8 +11,6 @@ public abstract class VisualStudioRazorParser
     {
         public abstract event EventHandler<DocumentStructureChangedEventArgs> DocumentStructureChanged;
 
-        public abstract RazorProjectEngine ProjectEngine { get; }
```

Comment:
Unrelated, just missed feedback from the previous PR.

Reply:
OK cool beans